rohd_devtools_extension/lib/rohd_devtools/cubit/rohd_service_cubit.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:devtools_app_shared/service.dart';
+import 'package:devtools_extensions/devtools_extensions.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:rohd_devtools_extension/rohd_devtools/models/tree_model.dart';
+import 'package:rohd_devtools_extension/rohd_devtools/services/tree_service.dart';
+
+part 'rohd_service_state.dart';
+
+class RohdServiceCubit extends Cubit<RohdServiceState> {
+  TreeService? treeService;
+
+  RohdServiceCubit() : super(RohdServiceInitial()) {
+    evalModuleTree();
+  }
+
+  Future<void> evalModuleTree() async {
```

Comment:
these two functions have a lot of identical code, can we refactor into some functions that they both use?